docs/content/multiple-ingress-controllers.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Multiple Ingress Controllers
+
+description: 
+weight: 100
+doctypes: ["concept"]
+toc: true
+---
+
+
+This doc is now available at https://docs.nginx.com/nginx-ingress-controller/installation/running-multiple-ingress-controllers/
```

Comment:
delete this doc and add an alias in the new file's frontmatter

Reply:
done